src/rails_new.rs

```diff
@@ -0,0 +1,31 @@
+use clap::Parser;
+
+#[derive(Parser)]
+#[command(version, about, long_about = None)]
+pub struct Cli {
+    #[clap(trailing_var_arg = true, required = true)]
+    pub args: Vec<String>,
+    #[clap(long, short = 'u', default_value = "3.2.3")]
+    pub ruby_version: Option<String>,
+    #[clap(long, short = 'r', default_value = "7.1.3")]
```

Comment:
If these have defaults, do they need to be `Option<String>` given that we call `.unwrap()` without actually checking for the `None` case?